test/unit/codegeneration/PureES6Transformer.js

```diff
@@ -70,7 +70,6 @@ suite('PureES6Transformer.js', function() {
     ].join('\n');
 
     var expected = [
-      '"use strict";',
```

Comment:
I still don't get it :(

The input is a `Module` and the output is a `Script` so we need to add `"use strict"` or we get the wrong semantics.